src/core_modules/capture-core/reducers/descriptions/currentSelections.reducerDescriptionGetter.js

```diff
@@ -213,5 +216,9 @@ export const getCurrentSelectionsReducerDesc = (appUpdaters: Updaters) => create
         categories: undefined,
         categoriesMeta: undefined,
     }),
+    [searchPageActionTypes.FALLBACK_SEARCH_COMPLETED]: (state, { payload: { trackedEntityTypeId } }) => ({
+        ...state,
+        trackedEntityTypeId,
+    }),
```

Comment:
Thinking abou this again. It seems like using the url to get this information and not the redux store might be a better idea. What you think?